typescript/core/.yarnrc

```diff
@@ -1,2 +0,0 @@
-version-tag-prefix "releases/typescript/core/v"
-version-git-message "TypeScript v%s"
```

Comment:
@jtbandes I did some brief searching and can't find the equivalent for these in yarn 3. Maybe we should switch to using npm to bump versions? Or have you used any equivalent in our yarn 3 repos?

Reply:
Using npm or yarn to bump versions + create tags only works if you have permissions to push directly to main. Generally it seems like that is no longer the case in many of our repos, so I don't know if we still need to bother with these.